lib/metrics.dart

```diff
@@ -152,6 +156,9 @@ class Metrics {
       'Cache': 'no-cache',
       'Content-Type': 'application/json',
       'Authorization': clientKey,
+      'X-UNLEASH-APPNAME': appName,
+      'X-UNLEASH-CONNECTION-ID': connectionId,
+      'X-UNLEASH-USER-AGENT': userAgent,
```

Comment:
Following Ivar's suggestion, yeah I think this would be better

```suggestion
      'X-UNLEASH-SDK-VERSION': userAgent,
```

x-unleash-sdk-info could also work

Reply:
I like x-unleash-sdk